File: backend/src/main/kotlin/com/example/researchreview/constants/ArticleStatus.kt
package com.example.researchreview.constants

enum class ArticleStatus(val value: Short){
    SUBMITTED(0), // Chờ xử lý
    REJECTED(1), // Bị từ chối
    PENDING_REVIEW(2), // chờ review
    IN_REVIEW(3), // đang review
    REVISIONS_REQUESTED(4), // yêu cầu sửa chữa
    REVISIONS(5), // đang sửa chữa
    ACCEPTED(6), // đã chấp nhận
    REJECT_REQUESTED(7), // Đề nghị loại bỏ
    ACCEPT_REQUESTED(8), // Đề nghị chấp thuận
}

File: frontend/src/constants/article-status.ts
export const ArticleStatus = {
  SUBMITTED: 'SUBMITTED', // Chờ xử lý
  REJECTED: 'REJECTED', // Bị từ chối
  PENDING_REVIEW: 'PENDING_REVIEW', // chờ review
  IN_REVIEW: 'IN_REVIEW', // đang review
  REVISIONS_REQUESTED: 'REVISIONS_REQUESTED', // yêu cầu sửa chữa
  REVISIONS: 'REVISIONS', // đang sửa chữa
  REJECT_REQUESTED: 'REJECT_REQUESTED', // đề nghị loại bỏ
  ACCEPT_REQUESTED: 'ACCEPT_REQUESTED', // đề nghị chấp thuận
  ACCEPTED: 'ACCEPTED', // đã chấp nhận
} as const;

export type ArticleStatusType = typeof ArticleStatus[keyof typeof ArticleStatus];